articles/tasks1.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Transfering and exporting Todoist tasks
+datatype: Tasks
+sources: Todoist
+destination: ["Google Sheet," "CSV", "JSON", "Things"]
+---
+
+# Todoist and Interoperability
+
+Todoist is lacking in the options available for exporting user data. The service does not allow exports in formats usable for other services and officially simply gives the only option of exporting data to Google Sheets.
+
+There are some third-party open source solutions that allow user information to be exported in JSON and CVS formats, but eventually, the information in these formats does not allow import compatibility in other similar services.
+
+## Todoist to Things
+
+Some services, such as Things, allow the user to migrate their information from Todoist using the Todoist public API (https://developer.todoist.com/sync/v9/#get-all-projects), but it can result in a complicated and tedious process for the user because queries must be performed to obtain the information requested.
+
+## Todoist to CSV or JSON
+This (third-party open source)[https://darekkay.com/todoist-export/] allows the user to backup Todoist data as CSV or JSON.
+
+* Once you select the format in which you want your data to be exported, you need to authorize and sign in with your Todoist account in order for the tool to begin the backup process.
```

Comment:
Does this require sharing a password with the tool?  If it does we should warn about that

Reply:
Yes, it does. I will write something to warn about it.

> Export for Todoist will be able to receive the following permissions:
> 
> View your user info
> Read your app data, including projects, tasks, notes, labels, and filters